docs/basics/primitives.mdx

```diff
@@ -95,6 +96,16 @@ The `updateValue` takes just one argument, which will be passed
 to the third argument of the write function of the atom.
 The behavior depends on how the write function is implemented.
 
+**Note:** as mentioned in the *atom* section, you have to take care of handling the reference of your atom, otherwise it may enter an infinite loop
+```typescript
+const safeAtom = atom(0)
+const Component = () => {
+  const [value] = useAtom(atom(0)) // Will cause an infinite loop
+  const [safeValue] = useAtom(safeAtmo) // is fine
```

Comment:
A typo. I would call it `stableAtom`. How does it sound?

Reply:
@dai-shi sounds better